config/ant-phase-verify.xml

```diff
@@ -39,9 +39,78 @@
     </checkstyle>
 
     <tstamp>
-      <format property="FINISHED" pattern="dd/MM/yyyy hh:mm:ss aa" />
+      <format property="FINISHED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
     </tstamp>
-    <echo>Checkstyle finished: ${FINISHED}</echo>
+    <echo>Checkstyle finished (checkstyle_checks.xml) : ${FINISHED}</echo>
+
+    <tstamp>
+      <format property="STARTED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
+    </tstamp>
+    <echo>Checkstyle started (checkstyle_non_main_files_checks.xml): ${STARTED}</echo>
+
+    <checkstyle config="config/checkstyle_non_main_files_checks.xml"
+                failOnViolation="true"
+                failureProperty="checkstyle.failure.property">
+      <path>
+        <fileset dir="${basedir}" includes="**/*">
+          <exclude name="src/it/resources/**/*" />
+          <exclude name="src/test/resources/**/*" />
+          <exclude name="src/test/resources-noncompilable/**/*"/>
```

Comment:
am I missing something? aren't these the 3 folders we wanted them to run on?
Please remove these 3 excludes.